### PR TITLE
Updating styles to match cloud.gov

### LINKED ISSUE #31: Bring global styling more inline with current cg-style practices.
Breaking out from #28 
- [x] Remove background image and style similar to https://cloud.gov/updates/
- [x] Add footer cloud.gov styling

### PR DESCRIPTION
**Pull request by juliasolorzano**
This pull request removes the blue background and matches the footer to match the one on cloud.gov. See screenshots below:

screen shot 2016-05-03 at 12 37 59 pm

screen shot 2016-05-03 at 12 38 12 pm

Fixes #31

### CODE REVIEW

**Comment by afeld** [comment]
Looks good! Just had one question here: https://github.com/18F/compliance-viewer/issues/31#issuecomment-216593424

File: assets/stylesheets/main.scss [hunk 1]
```diff
@@ -1,12 +1,9 @@
-
-$cg-blue: #1f2c38;
-
 .usa-alert-all-passing {
   background-image: none;
 }
 
 .usa-disclaimer {
```
Comment (msecret) [inline assets/stylesheets/main.scss:5]: In this main.scss file a custom CSS file on compliance viewer to override things from WDS and cg-style?

File: assets/stylesheets/main.scss [hunk 2]
```diff
@@ -18,14 +15,14 @@ $cg-blue: #1f2c38;
...
-  width: auto;
   float: left;
-}
+  width: auto;
 
-.logo h1 a {
-  text-decoration: none;
-  font-size: 2rem;
-  color: #595959;
+  h1 a {
```
Comment (msecret) [inline assets/stylesheets/main.scss]: Does this need the `h1` or does just `a` work?

Is this something that should be added to cg-style? I think the rule kinda makes sense there, I can add.

File: assets/stylesheets/main.scss [hunk 3]
```diff
@@ -18,14 +15,14 @@ $cg-blue: #1f2c38;
...
+    color: #595959;
```
Comment (msecret) [inline assets/stylesheets/main.scss:22]: Is this a separate color from any of the cg-style colors?

File: assets/stylesheets/main.scss (hunk 1 above)
Comment (juliasolorzano) [inline assets/stylesheets/main.scss:5]: This is a custom file for compliance viewer.

File: assets/stylesheets/main.scss (hunk 2 above)
Comment (juliasolorzano) [inline assets/stylesheets/main.scss]: +1 on changing to just `<a>`

**Pull request merged**